.github/workflows/test-notebooks.yml

```diff
@@ -88,7 +88,9 @@ jobs:
           rapids-logger "nvidia-smi"
           nvidia-smi
       - name: Test notebooks
-        run: /home/rapids/test_notebooks.py -i /home/rapids/notebooks -o /home/rapids/notebooks_output
+        run: |
+          . /opt/conda/etc/profile.d/conda.sh; conda activate rapids
```

Comment:
With the environment hacking is this necessary?

Reply:
Ah you're right, it shouldn't be! I'll try reverting it.